### DISCUSSION TITLE
Support digest auth?

### DISCUSSION THREAD

**Discussion by shiyuhang0** [node-fetch/node-fetch#1732] [Ideas]
Does node-fetch support digest auth?

**Comment by LinusU** [comment]
Is this something that browsers support? Is it part of the official Fetch specification?

I can't find the word "digest" in [the spec](https://fetch.spec.whatwg.org), so I believe that the answer is: No, and support will not be added unless it's added to the upstream Fetch specification.

**Reply by shiyuhang0** [reply]
get
